### PR TITLE
Added hubblesite API

### PR DESCRIPTION
**Pull request by AderbalFarias**
Thank you for taking the time to work on a Pull Request for this project!

To ensure your PR is dealt with swiftly please check the following:

- [ ] Your submissions are formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [ ] Your additions are ordered alphabetically
- [ ] Your submission has a useful description
- [ ] The description does not end with punctuation
- [ ] Each table column should be padded with one space on either side
- [ ] You have searched the repository for any relevant issues or pull requests
- [ ] Any category you are creating has the minimum requirement of 3 items
- [ ] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

**Comment by marekdano** [comment]
Thanks for the contribution @AderbalFarias!!!
Can you please resolve the conflict?

**Comment by AderbalFarias** [comment]
Conflict resolved

File: README.md [hunk 1]
```diff
@@ -746,7 +746,9 @@ API | Description | Auth | HTTPS | CORS |
 |---|---|---|---|---|
 | [arcsecond.io](https://api.arcsecond.io/) | Multiple astronomy data sources | No | Yes | Unknown |
 | [CORE](https://core.ac.uk/services#api) | Access the world's Open Access research papers | `apiKey` | Yes | Unknown |
+| [GBIF](http://api.gbif.org/v1/) | Global Biodiversity Information Facility | No | Yes | Yes |
```
Comment (marekdano) [inline README.md:749]: I think this API is the same comparing with the one below. 
Can you please remove this record?

File: README.md [hunk 2]
```diff
@@ -746,7 +746,9 @@ API | Description | Auth | HTTPS | CORS |
...
 | [GBIF](https://www.gbif.org/developer/summary) | Global Biodiversity Information Facility | No | Yes | Yes |
+| [hubblesite](http://hubblesite.org/api/documentation/) | Space Telescope News Releases | No | No | Unknown |
```
Comment (marekdano) [inline README.md:751]: Can you please change `http` to `https`?

I can also see the API support the `HTTPS`. Can you please change `No` to `Yes`?

**Review by marekdano — CHANGES_REQUESTED** [review]
Thanks for resolving the conflict!
I left some comments.

File: README.md (hunk 1 above)
Comment (AderbalFarias) [reply README.md]: Duplication removed

**Review by marekdano — APPROVED** [review]

**Pull request merged**